pulsar-broker/src/test/java/org/apache/pulsar/broker/service/RackAwareTest.java

```diff
@@ -50,10 +57,8 @@ public RackAwareTest() {
         super(0);
     }
 
-    @BeforeClass(alwaysRun = true)
-    protected void setup() throws Exception {
```

Comment:
(probably it is okay, but I want to double check) 
why are you removing this ?

Reply:
@lhotari Lari gives some ideas here [#11349 (comment)](https://github.com/apache/pulsar/issues/11349#issuecomment-882449466)

Reply:
@eolivelli the initialization of the test class doesn't happen in a controlled way unless this is changed.

Reply:
This is fine.